R/object-defaults.R

```diff
@@ -29,6 +29,20 @@ object_defaults.data <- function(x) {
 }
 
 #' @export
+object_defaults.package <- function(x) {
+  desc <- read.description(file.path(x$value$base_path, "DESCRIPTION"))
+
+  pkg_name <- x$value$pkg_name
+  list(
+    docType = "package",
+    title = as.character(desc$Title),
+    description = as.character(desc$Description),
+    aliases = c("NULL", paste0(pkg_name, "-package")), # See #202
```

Comment:
Could you make the comment a bit more direct? i.e. "NULL" prevents addition of default aliases